src/_playground/Routes.js

```diff
@@ -400,12 +402,14 @@ export class Routes extends Component {
                         </header>
                         <div className={sideBarClasses}>
                             <div className='frDocs-Search'>
-                                <InputGroup
-                                    glyph='search'
-                                    inputPlaceholder='Search'
-                                    inputProps={{ 'aria-label': 'Search' }}
-                                    inputValue={this.state.query}
-                                    onChange={this.onChangeHandler} />
+                                <InputGroup type='icon'>
+                                    <InputGroup.Addon>
+                                        <Icon glyph='search' />
+                                    </InputGroup.Addon>
+                                    <FormInput
+                                        onChange={this.onChangeHandler} placeholder='Search'
```

Comment:
Very minor nit:  when stacking props on multiple lines, there should only be one per line -- `placeholder` should be on its own line.